### ISSUE TITLE
Edit submission not rendering form tags

### ISSUE THREAD

**Issue by StuartMcD** [solspace/craft3-freeform#1047]
OK, upgrading all the things and in one area of the site I have the ability to edit submissions. In this upgrade, changing tags it renders the fields and js but no opening/closing form tags.

**Steps to reproduce**

1. Using a token in a segment, capture and pass to 'form' tag
2. Inputs represented on screen with correct info
3. No <form> tag renders
When I use the same tags without 'token' attribute, it's fine. Soon as you pass token the outer form tags fail to render :(

- Craft Version: 4.7.2.1
- Freeform Version: 5.0.10
- Freeform Edition: Pro
- Fresh Install or Upgrade: Upgrade

**Additional context**

**Comment by kjmartens** [comment]
Hi @StuartMcD,

What does your URL (to edit the form submission) and the template code look like? 🙂

**Comment by StuartMcD** [comment]
The form 'guts' render but no form wrap

https://soccer.test/player/update?pid=[token]
```

{% set uniqueID = craft.app.request.getParam('pid') %}
{% set registerForm = seasonParamters.currentSignupForm.handle %}

{% set form = freeform.form( registerForm, {
	 attributes: {
		class:"form",
		id:"player_form",
	 },
	disable: ["submitButtons"],
	submissionToken: uniqueID|default(null),
	})
%}


{{ form.renderTag({
	returnUrl: "/player/update?pid={{ submission.token }}"}) 
}}
```

![image](https://github.com/solspace/craft-freeform/assets/5855813/53262376-2f3c-444e-8a4b-bc7e62a6fd96)

**Comment by kjmartens** [comment]
Hi @StuartMcD,

Hmm, that is very weird. I tried testing this several ways, and I cannot reproduce the issue.

What happens if you try editing submissions in the Freeform demo templates? If you don't want to install them, can you create a brand new template and include the following:

``` twig
<html>
<head></head>
<body>

{% if craft.app.request.getQueryParam('edit') %}
    {% set submissionToken = craft.app.request.getQueryParam('edit') %}
{% endif %}

{{ freeform.form("yourFormHandle", {
    formattingTemplate: 'basic-light/index.twig'
    submissionToken: submissionToken|default(null),
}).render() }}

</body>
</html>
```

Then access it with the applicable URL structure. 🙂

**Comment by StuartMcD** [comment]
Had errors turned off previous - seeing these errors when viewing certain forms but not others.

This is the error returned when using my default template:
![image](https://github.com/solspace/craft-freeform/assets/5855813/9b8fa628-2ddd-4c7e-9a0a-04a9c11524e2)


When using the demo templates, same form, if I click view submissions I get:
![image](https://github.com/solspace/craft-freeform/assets/5855813/7dccaf45-c0c7-4e06-9203-40866d4d0a10)

Yet on some simpler forms it all works as expected.

**Comment by StuartMcD** [comment]
On upgrade - in this one form, a field that was textarea has become a radio field. Trying to edit an entry in some forms works fine, yet in the other forms when editing - the CP gives me this error:

![image](https://github.com/solspace/craft-freeform/assets/5855813/2e904445-56f9-4c3e-8c70-0270b053799e)

craft_dev_mode = true

**Comment by kjmartens** [comment]
I'm sorry for the trouble. It sounds like this was likely caused by issues with the migration then. We addressed several issues and released an update today (5.0.11). Are you able to revert back to 4.x and then run the migration again?

**Comment by StuartMcD** [comment]
OK, finally had a good session to try and figure it out. Seems in a series of forms I had a field that was likely a left over from another time that didnt do anything and didnt contain anything. Once removed from the form, all entries (CP and front-end) all worked inc. allowed editing, ie. rendered with <form> tags  🎉

Other forms that also displayed errors in CP, appear to be using the same field/handle from another form, once I changed some of them it worked fine.
